src/bootstrap/install.rs

```diff
@@ -32,7 +32,7 @@ fn install_sh(
     builder.info(&format!("Install {} stage{} ({:?})", package, stage, host));
 
     let prefix = default_path(&builder.config.prefix, "/usr/local");
-    let sysconfdir = prefix.join(default_path(&builder.config.sysconfdir, "/etc"));
+    let sysconfdir = prefix.join(default_path(&builder.config.sysconfdir, "etc"));
```

Comment:
I think this would change the default from `/etc` to `/usr/local/etc`.  

To retain the original behavior, I think it would need to be restructured to handle the case when `sysconfdir` is not set.

Reply:
Yeah I just thought that using `/etc` as the default wouldn't work well since it would require some sort of intervention to resolve the permission issue. But if that's indeed what people here want to do, then yes, a bit of restructuring would be necessary.

Reply:
Oops, I take back what I said. It looks like `/usr/local/etc` would also require permission resolution, generally speaking (on my Mac it doesn't, but on my Linux it does). That makes me wonder---is it intentional to always use `/etc`?

Reply:
It is intentional that the default is `/etc` (documented [here](https://github.com/rust-lang/rust/blob/a84d1b21aea9863f0fc5f436b4982d145dade646/config.toml.example#L318-L320)), and should use prefix if `sysconfdir` is relative.

Reply:
Got it. My bad! I'll just close this pull request.